ddtrace/_monkey.py

```diff
@@ -202,10 +202,7 @@ def patch(raise_errors=True, **patch_modules):
     )
 
 
-@deprecated(
-    message="This function will be removed.",
-    version="1.0.0",
-)
+@remove(removal_version="1.0.0")
```

Comment:
``removed`` annotation will log the following warning: `Using function/method patch_module is deprecated and will be removed in version '1.0.0'`